rest-api/src/main/resources/paths/studies/v5_studies_studyId_participants_demographics_validation_categoryName.yml

```diff
@@ -0,0 +1,84 @@
+post:
+    operationId: saveDemographicsValidationConfig
+    summary: Save a DemographicValuesValidationConfig
+    description: Save a DemographicValuesValidationConfig to configure demographics validation.
+    tags:
+        - Study Participants
+        - Demographics
+        - _For Study Coordinators
+        - _For Researchers
```

Comment:
Change _For Study Coordinators to _For Study Designers and change _For Researchers to _For Developers.

In both POST and DELETE.